src/ExadelMentorship.WebApi/Program.cs

```diff
@@ -15,11 +16,12 @@
 builder.Services.AddSwaggerGen();
 
 builder.Services.AddBlServices();
-
+builder.Services.AddTransient<ExceptionHandlingMiddleware>();
```

Comment:
Does an ASP.NET Core middleware need to be registered in DI?

Reply:
<img width="628" alt="1" src="https://user-images.githubusercontent.com/59729680/171001024-0e1d5990-156f-4c13-b8de-39461dc4655f.png">

Reply:
Oh, I see. I got confused with `app.UseExceptionHandler` from the [docs](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/error-handling?view=aspnetcore-6.0#exception-handler-lambda). Because it didn't have any exception middleware registered explicitly. I believe `WebApplication.CreateBuilder` registers its own [exception middelware](https://github.com/dotnet/aspnetcore/blob/main/src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerMiddleware.cs).

So, if there is already an exception middleware, then there is no point creating our own one. Try utilizing `app.UseExceptionHandler` instead plz.

Reply:
done